### PR TITLE
doc: change wording of PR guidance

### PR DESCRIPTION
**Pull request by kartikgajjar7**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: docs/how-to-open-a-pull-request.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We recommend using [conventional title and messages](https://www.conventionalcom
 >
 > `fix(learn): tests for the do...while loop challenge`
 
-When opening a Pull Request(PR), you can use the below to determine the type, scope (optional), and description.
+Whenever you opening a Pull Request(PR), you can use the below to determine the type, scope (optional), and description.
```
Comment (Sboonny) [inline docs/how-to-open-a-pull-request.md:18]:
```suggestion
Whenever you open a Pull Request(PR), you can use the below to determine the type, scope (optional), and description.
```

File: docs/how-to-open-a-pull-request.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ You can select a scope from [this list of labels](https://github.com/freeCodeCam
 
 **Description:**
 
-Keep it short (less than 30 characters) and simple, you can add more information in the PR description box and comments.
+Keep it short,to the point (less than 30 characters) and simple, you can add more information in the PR description box and comments.
```
Comment (Sboonny) [inline docs/how-to-open-a-pull-request.md:35]:
```suggestion
Keep it short, to the point (less than 30 characters) and simple, you can add more information in the PR description box and comments.
```

File: docs/how-to-open-a-pull-request.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ Some examples of good PR titles would be:
 
 3. Submit the pull request from your branch to freeCodeCamp's `main` branch.
 
-4. In the body of your PR include a more detailed summary of the changes you made and why.
+4. In the body of your PR include a more detailed summary of the changes you made and why you changed.
```
Comment (Sboonny) [inline docs/how-to-open-a-pull-request.md:58]:
```suggestion
4. In the body of your PR, include a more detailed summary of the changes you made and how your changes are helpful.
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
I don't think they are typos, but thank you for trying to improve them

**Review by Sboonny — DISMISSED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Kartik2220, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Comment by Kartik2220** [comment]
give me improvements suggestions if needed :)

**Comment by Sboonny** [comment]
> give me improvements suggestions if needed :)

@Kartik2220 sure. 

Mentioning the old change was worse than the current one, and following our pr title guidelines will make maintainers take you more seriously

File: docs/how-to-open-a-pull-request.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ You can select a scope from [this list of labels](https://github.com/freeCodeCam
 
 **Description:**
 
-Keep it short (less than 30 characters) and simple, you can add more information in the PR description box and comments.
+Keep it short (less than 30 characters), to the point and simple, you can add more information in the PR description box and comments.
```
Comment (jeremylt) [inline docs/how-to-open-a-pull-request.md:35]:
```suggestion
Keep it short (less than 30 characters) and simple; you can add more information in the PR description box and comments.
```

File: docs/how-to-open-a-pull-request.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ Some examples of good PR titles would be:
 
 3. Submit the pull request from your branch to freeCodeCamp's `main` branch.
 
-4. In the body of your PR include a more detailed summary of the changes you made and why.
+4. In the body of your PR, include a more detailed summary of the changes you made and how your changes are helpful.
```
Comment (jeremylt) [inline docs/how-to-open-a-pull-request.md:58]:
```suggestion
4. Include a more detailed summary of the changes you made and how your changes are helpful in the body of your PR.
```

**Review by jeremylt — COMMENTED** [review]
If we are doing to change the wording around, we should actually make changes for grammatical correctness.

Just changing the wording isn't value added, IMHO.

**Comment by jeremylt** [comment]
Side note - your PRs are not using the documentation that you are trying to edit. The titles don't match the recommended format.

**Comment by Kartik2220** [comment]
> Side note - your PRs are not using the documentation that you are trying to edit. The titles don't match the recommended format.

can you explain me brifly what i have to improve ?

**Comment by jeremylt** [comment]
> can you explain me brifly what i have to improve ?

https://github.com/freeCodeCamp/freeCodeCamp/blob/972dbd5c27579ffc77c10acc6fec8170943cf1dd/docs/how-to-open-a-pull-request.md?plain=1#L8

**Comment by Kartik2220** [comment]
> > can you explain me brifly what i have to improve ?
> 
> https://github.com/freeCodeCamp/freeCodeCamp/blob/972dbd5c27579ffc77c10acc6fec8170943cf1dd/docs/how-to-open-a-pull-request.md?plain=1#L8

i changed typo to FIX is it okay ?

**Comment by jeremylt** [comment]
You probably want `doc` since you are changing the documentation, like it says here:
https://github.com/freeCodeCamp/freeCodeCamp/blob/972dbd5c27579ffc77c10acc6fec8170943cf1dd/docs/how-to-open-a-pull-request.md?plain=1#L27

**Comment by Kartik2220** [comment]
> You probably want `doc` since you are changing the documentation, like it says here:
> 
> https://github.com/freeCodeCamp/freeCodeCamp/blob/972dbd5c27579ffc77c10acc6fec8170943cf1dd/docs/how-to-open-a-pull-request.md?plain=1#L27



> You probably want `doc` since you are changing the documentation, like it says here:
> 
> https://github.com/freeCodeCamp/freeCodeCamp/blob/972dbd5c27579ffc77c10acc6fec8170943cf1dd/docs/how-to-open-a-pull-request.md?plain=1#L27

YES DONE

**Comment by jeremylt** [comment]
Thanks for the fix. 

There's no need to scream in ALL CAPS.

**Review by jeremylt — APPROVED** [review]

**Review by Sboonny — APPROVED** [review]

**Pull request merged**